app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player, ABC
+
+
+class Dwarf(Player, ABC):

```

Comment:
You can select inheritance only from Player, because Player class already inherit from ABC.
Also delete from import ABC.
So you should fix this issue in all files

Reply:
If we do this way we have warning about "Where is @abstractmethod?"